controller_manager/doc/parameters_context.yaml

```diff
@@ -25,3 +25,7 @@ diagnostics.threshold.hardware_components.periodicity: |
 
 diagnostics.threshold.hardware_components.execution_time.mean_error: |
   The ``execution_time`` diagnostics will be published for all hardware components. The ``mean_error`` for a synchronous hardware component will be computed against zero, as it should be as low as possible. However, the ``mean_error`` for an asynchronous hardware component will be computed against its desired read/write period, as the hardware component can take a maximum of the desired period to execute the read/write cycle.
+
+enforce_command_limits: |
```

Comment:
why adding this here instead of the parameter description itself? now it is rendered like this:

![image](https://github.com/user-attachments/assets/cd33f512-2c37-40c8-bda8-6bd231f8813d)